cockroach-commands.md

```diff
@@ -23,6 +22,13 @@ Command | Usage
 
 ## Logging Flags
 
+By default, CockroachDB logs all messages and errors to files (see `--log-dir`) and copies errors with a severity of `ERROR` or higher to the standard error stream (see `--alsologtostderr`). 
+
 Flag | Description
 -----|------------
- | 
+`--alsologtostderr` | Copy log messages at or above this severity level to the standard error stream in addition to log files. Possible values: `INFO`, `WARNING`, `ERROR`, and `FATAL`. <br><br>**Default:** `ERROR`
```

Comment:
No, the `{INFO,WARNING,ERROR,FATAL}` terms do not need to be in all caps. Any capitalization is ok.